app/controllers/books_suggestions_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class BooksSuggestionsController < ApplicationController
```

Comment:
You shouldn't be inheriting from `ApplicationController` to skip validation, because you will probably need many other things. You can use `skip_before_filter` so it doesn't authenticate the user